test/mockupdb/test_op_msg_read_preference.py

```diff
@@ -159,11 +159,10 @@ def test(self):
             expected_pref = pref
         else:
             self.fail('unrecognized op_type %r' % operation.op_type)
-
         # For single mongod we send primaryPreferred instead of primary.
-        if expected_pref == ReadPreference.PRIMARY and self.single_mongod:
+        if (expected_pref == ReadPreference.PRIMARY and self.single_mongod
+                and operation.name != "command"):
```

Comment:
Why is this needed?

Reply:
That might be a pymongo bug. Can you open a new ticket to investigate?

Reply:
Done.